week-4/E-arrays-of-objects/exercise-3.js

```diff
@@ -60,13 +60,43 @@ var restaurantFinderApplication = {
     applicationVersion: "1.0",
     restaurants: restaurants,
     findAvailableRestaurants: function(numberOfPeople) {
+       
+        return restaurants.filter(getAvailableRestaurants).map(getName);
+
+        function getAvailableRestaurants(restaurant){
```

Comment:
Well done, this works.  However, the function you have named `getAvailableRestaurants` does not get available restaurants.   It returns a boolean (true or false), not any restaurants.  And it takes ONE restaurant to inspect.

Can you think of a better name for this function, based on what it does and what it returns?  Hint: it is a predicate function, which we covered briefly in week 3.